### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by coderschoolreview** [alroseylleno/ML01-Week01#1]
## Goal of this Assignment
The goal of this assignment was to introduce you to 2 main concepts in Machine Learning: 
- Data Visualization
- Exploratory Data Analysis. 

You learn how to query and clean data using pandas library in Python, make some plots which help to understand more about data with Seaborn library.

## Things you did well:
- You did not only accomplished every single problem but also add more advanced question. This is out of my expectation and is very encouraged. You did really great :D !!
- You practice the Python code very well ! 
- You can make use of different Pandas indexing operators.
- You make use of new Pandas function :D, which is `nlargest()`. This is remarkable :D

## Things to work on:
- You can solve all the problems, which is really great. The thing you need to improve is that there are some questions that can be answered much more easily and simply :D. Don't make things too complicated :D.

## To sum up
- What you did is very great and exceed our team's expectation :D. 
- Keep up the spirit and practice more, you can improve much faster !!
